helm/cert-manager-app/templates/controller-deployment.yaml

```diff
@@ -47,6 +47,10 @@ spec:
           - --default-issuer-kind={{ .Values.controller.defaultIssuer.kind }}
           - --default-issuer-group={{ .Values.controller.defaultIssuer.group }}
           {{- end }}
+          {{- if eq .Values.Installation.V1.GiantSwarm.CertManager.AcmeSolver "dns01" }}
+          - --dns01-recursive-nameservers-only
+          - --dns01-recursive-nameservers="{{ .Values.Installation.V1.GiantSwarm.CertManager.AcmeSolver.DNSServer }}:53"
```

Comment:
```suggestion
          - --dns01-recursive-nameservers="{{ .Values.Installation.V1.GiantSwarm.CertManager.DNSServer }}:53"
```

Reply:
it should be on the same level as the acmesolver (see https://github.com/giantswarm/installations/pull/1775/files#diff-9f1254843a71b22c5da04204498cb62f77979e02c26393cd328b58e0b5e0f908R21)

Reply:
I agree :) Both should be under `CertManager`. But it is not the case: `.. .CertManager.AcmeSolver.DNSServer`

Reply:
oh sorry i'm reading the diff the wrong way around :facepalm: